src/components/drawer/styles/Drawer.styles.ts

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import { DrawerProps } from '../content/drawer_types';
+import { Position } from '../utils/constants';
+export const OverlayLeftRight = styled.div<DrawerProps>(
+  ({ theme, visible, position, width, height }) => ({
+    backgroundColor: theme.palette.secondary.main,
+    position: 'absolute',
```

Comment:
chera absolute ?

Reply:
chun overlay daqiqan az bala chasbide be goshe baz she bar daram kharab mishe